### ISSUE TITLE
Display the original commit message when mirroring

### ISSUE THREAD

**Issue by minwoox** [line/centraldogma#710]
Currently, the commit message of a commit that is pushed by the mirroring task looks like this:
```
Mirror fbee605, git+ssh://github.com/xxx.git#master to the repository 'foo'
```
It does not contain the original commit message which is useful to users. It would be nice if we can display the original commit message.
We can retrieve the message via:
```java
Iterable<RevCommit> call = git.log().setMaxCount(1).call();
```

**Comment by minwoox** [comment]
We might display the message using the [detail](https://github.com/line/centraldogma/blob/master/server/src/main/java/com/linecorp/centraldogma/server/command/AbstractPushCommand.java#L79-L81) field.
However, the current UI does not display it so we should revamp the UI beforehand.

**Comment by trustin** [comment]
There might be more than 1 commit involved in a mirroring attempt. Should we show every commit message (up to N commits)?

**Comment by minwoox** [comment]
> Should we show every commit message (up to N commits)?

That's a good idea.

**Comment by trustin** [comment]
At second thought, we probably should only show the latest one only, given we're going to use shallow clone and shallow fetch (see #801).

**Comment by thachlp** [comment]
From my understanding, here is the pr: https://github.com/line/centraldogma/pull/818
Please help review 🤗

**Comment by trustin** [comment]
Closed via #818 Thanks, @thachlp!
